components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1543,9 +1543,12 @@ void RewardsServiceImpl::TriggerOnGetPublisherActivityFromUrl(
     ledger::Result result,
     std::unique_ptr<ledger::PublisherInfo> info,
     uint64_t windowId) {
-  for (auto& observer : private_observers_)
-    observer.OnGetPublisherActivityFromUrl(this, result, std::move(info),
-                                           windowId);
+  if (!info) {
```

Comment:
never mind, I see now that it was moved here

Reply:
yep. maybe i should assert though, to play it safe?

Reply:
still a bit strange though and maybe we should look at this in a follow-up. Not sure why we'd create a new one instead of just passing in `nullptr`, but maybe there's some other reason

Reply:
i think it's done this way in order to get the appropriate default initialization for the struct members, otherwise you'd have that code duplicated in ledger and rewards. that was my take anyway. agreed that it's strange.